src/Schema/Events/Click.ts

```diff
@@ -207,6 +207,24 @@ export interface ClickedAuctionResultItem extends ClickedEntityGroup {
   expanded: boolean
 }
 
+/**
+ * A user clicks the Browse Similar Artworks button on an closed auction artwork page
+ *
+ * This schema describes events sent to Segment from [[clickedEntityGroup]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "clickedBrowseSimilarArtworks",
+ *    context_module: "artworkClosedLotHeader",
+ *    context_page_owner_type: "artwork",
+ *  }
+ * ```
+ */
+export interface ClickedBrowseSimilarArtworks extends ClickedEntityGroup {
+  acttion: ActionType.clickedBrowseSimilarArtworks
+}
```

Comment:
**Suggetion**: Should we add `context_module` and `context_page_owner_type` here as well?

Reply:
nvm, I didn't notice that it was inside Click 😅

Reply:
My bad my bad! Fixed, pushing changes